# https://github.com/uutils/coreutils/commit/03fc5ca381d26b292134a1d5b13116dae1a16525

--- a/src/uu/chmod/Cargo.toml
+++ b/src/uu/chmod/Cargo.toml
@@ -19,7 +19,7 @@ path = "src/chmod.rs"
 [dependencies]
 clap = { workspace = true }
 libc = { workspace = true }
-uucore = { workspace = true, features = ["fs", "mode", "perms"] }
+uucore = { workspace = true, features = ["entries", "fs", "mode", "perms"] }
 
 [[bin]]
 name = "chmod"
