--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@ ui_test = []
 [dependencies]
 anyhow = { version = "1.0.86", features = ["backtrace"] }
 async-trait = "0.1.81"
-chrono = "0.4"
+chrono = "0.4.38"
 clap-verbosity-flag = "2.2.1"
 clap = { version = "4.5.13", features = ["derive"] }
 crossterm = "0.27.0"
@@ -77,3 +77,8 @@ strum = { version = "0.26.3", features =
 [target.'cfg(target_os = "windows")'.build-dependencies]
 http_req = "0.11.1"
 zip = "2.1.6"
+
+[patch.crates-io]
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.38" }
+iana-time-zone = { path = "../iana-time-zone-0.1.60" }
