--- a/vendor/openssl-probe-0.1.5/.cargo-checksum.json
+++ b/vendor/openssl-probe-0.1.5/.cargo-checksum.json
@@ -1 +1 @@
-{"files":{"Cargo.lock":"ad8c068c0dffdd9d57068d1e8ddb5cb0354070e06087bef5b37fdda87b1a9ef9","Cargo.toml":"386e7ac7a1a7bc851e9e611e14396b6b3ead5bea731fac8866bd5db79f77943e","LICENSE-APACHE":"a60eea817514531668d7e00765731449fe14d059d3249e0bc93b36de45f759f2","LICENSE-MIT":"378f5840b258e2779c39418f3f2d7b2ba96f1c7917dd6be0713f88305dbda397","README.md":"92bce73b5b1e41134e103fc07974aeab78866c7e0335a613436d3de85f0f8d04","examples/probe.rs":"78fd7274bd050c3d53b0c855be121d2388d4b8ac2fa0e52160a55ddc65e2c2f0","src/lib.rs":"7915d920e7e388586a93376fc8860b4eb5117da2f53f78d7bffa4e08e9f3426a"},"package":"ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"}
\ No newline at end of file
+{"files":{"Cargo.lock":"ad8c068c0dffdd9d57068d1e8ddb5cb0354070e06087bef5b37fdda87b1a9ef9","Cargo.toml":"386e7ac7a1a7bc851e9e611e14396b6b3ead5bea731fac8866bd5db79f77943e","LICENSE-APACHE":"a60eea817514531668d7e00765731449fe14d059d3249e0bc93b36de45f759f2","LICENSE-MIT":"378f5840b258e2779c39418f3f2d7b2ba96f1c7917dd6be0713f88305dbda397","README.md":"92bce73b5b1e41134e103fc07974aeab78866c7e0335a613436d3de85f0f8d04","examples/probe.rs":"78fd7274bd050c3d53b0c855be121d2388d4b8ac2fa0e52160a55ddc65e2c2f0","src/lib.rs":"366f022d6a888c2c5a93a10fc79c49303be98f62183ade225dea6326cd6befb1"},"package":"ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"}
\ No newline at end of file
--- a/vendor/openssl-probe-0.1.5/src/lib.rs
+++ b/vendor/openssl-probe-0.1.5/src/lib.rs
@@ -24,6 +24,7 @@ pub fn find_certs_dirs() -> Vec<PathBuf>
 fn cert_dirs_iter() -> impl Iterator<Item = &'static Path> {
     // see http://gagravarr.org/writing/openssl-certs/others.shtml
     [
+        "/opt/etc/ssl", // Entware
         "/var/ssl",
         "/usr/share/ssl",
         "/usr/local/ssl",
@@ -37,7 +38,6 @@ fn cert_dirs_iter() -> impl Iterator<Ite
         "/etc/pki/tls",
         "/etc/ssl",
         "/etc/certs",
-        "/opt/etc/ssl", // Entware
         "/data/data/com.termux/files/usr/etc/tls",
         "/boot/system/data/ssl",
     ]
--- a/vendor/openssl-src-111.28.2+1.1.1w/src/lib.rs
+++ b/vendor/openssl-src-111.28.2+1.1.1w/src/lib.rs
@@ -229,6 +229,15 @@ impl Build {
         }
 
         let os = match target {
+            "aarch64-openwrt-linux-gnu" => "linux-aarch64", // Entware/OpenWrt
+            "arm-openwrt-linux-gnueabi" => "linux-armv4", // Entware/OpenWrt
+            "armv5te-openwrt-linux-gnueabi" => "linux-armv4", // Entware/OpenWrt
+            "armv7-openwrt-linux-gnueabi" => "linux-armv4", // Entware/OpenWrt
+            "i586-openwrt-linux-gnu" => "linux-elf", // Entware/OpenWrt
+            "i686-openwrt-linux-gnu" => "linux-elf", // Entware/OpenWrt
+            "mips-openwrt-linux-gnu" => "linux-mips32", // Entware/OpenWrt
+            "mipsel-openwrt-linux-gnu" => "linux-mips32", // Entware/OpenWrt
+            "x86_64-openwrt-linux-gnu" => "linux-x86_64", // Entware/OpenWrt
             "aarch64-apple-darwin" => "darwin64-arm64-cc",
             // Note that this, and all other android targets, aren't using the
             // `android64-aarch64` (or equivalent) builtin target. That
