--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,7 +63,7 @@ anyhow = "1.0"
 async-compression = { version = "0.4", default-features = false, optional = true, features = ["brotli", "deflate", "gzip", "zstd", "tokio"] }
 bcrypt = { version = "0.15", optional = true }
 bytes = "1.5"
-chrono = { version = "0.4", default-features = false, features = ["std", "clock"], optional = true }
+chrono = { version = "=0.4.37", default-features = false, features = ["std", "clock"], optional = true }
 clap = { version = "4.5", features = ["derive", "env"] }
 form_urlencoded = "1.2"
 futures-util = { version = "0.3", default-features = false }
@@ -138,3 +138,8 @@ pre-build = [
     "rm base.tar.xz",
     "rm -rf /tmp/netbsd",
 ]
+
+[patch.crates-io]
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.37" }
+iana-time-zone = { path = "../iana-time-zone-0.1.60" }
