--- a/pueue_lib/Cargo.toml
+++ b/pueue_lib/Cargo.toml
@@ -18,7 +18,7 @@ maintenance = { status = "actively-devel
 anyhow = "1.0"
 async-trait = "0.1"
 byteorder = "1.5"
-chrono = { version = "0.4", features = ["serde"] }
+chrono = { version = "0.4.34", features = ["serde"] }
 dirs = "5.0"
 rand = "0.8"
 rcgen = "0.12"
@@ -51,6 +51,9 @@ better-panic = { workspace = true }
 pretty_assertions = { workspace = true }
 tokio = { workspace = true }
 
+# mips: error: failed to run custom build command for `ring v0.17.7`
+ring = { workspace = true }
+
 # --- Platform specific dependencies ---
 # Windows
 [target.'cfg(windows)'.dependencies]
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@ edition = "2021"
 rust-version = "1.67"
 
 [workspace.dependencies]
-chrono = { version = "0.4", features = ["serde"] }
+chrono = { version = "0.4.34", features = ["serde"] }
 command-group = "2"
 log = "0.4"
 serde = "1.0"
@@ -32,5 +32,13 @@ anyhow = "1"
 better-panic = "0.3"
 pretty_assertions = "1"
 
+# mips: error: failed to run custom build command for `ring v0.17.7`
+ring = "0.17.8"
+
 [profile.release]
 lto = "thin"
+
+[patch.crates-io]
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.34" }
+iana-time-zone = { path = "../iana-time-zone-0.1.60" }
