--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,3 +57,10 @@ default-features = false
 [workspace.dependencies.sqlx]
 version = "=0.7.3"
 features = ["runtime-tokio-rustls", "time", "postgres", "uuid"]
+
+[patch.crates-io]
+# armv5te/mips/mipsel (Atomic*64)
+sqlx = { path = "../sqlx-0.7.3" }
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.37" }
+iana-time-zone = { path = "../iana-time-zone-0.1.60" }
