--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ sha2 = "0"
 bincode = "1"
 bytefmt = "0"
 capnp = "0"
-chrono = { version = "0", default-features = false, features = ["clock", "serde", "std"] }
+chrono = { version = "=0.4.40", default-features = false, features = ["clock", "serde", "std"] }
 chrono-tz = { version = "0", features = ["serde"] }
 clap = { version = "4", features = ["wrap_help", "derive", "env", "string"] }
 clap_complete = "4"
@@ -152,3 +152,8 @@ harness = false
 name = "encstr"
 path = "benches/encstr/benches.rs"
 harness = false
+
+[patch.crates-io]
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.40" }
+iana-time-zone = { path = "../iana-time-zone-0.1.61" }
