--- a/src/cli.rs
+++ b/src/cli.rs
@@ -25,7 +25,7 @@ EXAMPLES:
   ox test.txt test2.txt
   ox /home/user/docs/test.txt
   ox -c config.lua test.txt
-  ox -r -c ~/.config/.oxrc -f lua my_file.lua
+  ox -r -c /opt/etc/.oxrc -f lua my_file.lua
   tree | ox -r --stdin\
 ";
 
@@ -66,7 +66,7 @@ impl CommandLineInterface {
             stdin: j.contains("--stdin"),
             file_type: j.option_arg::<String, Key>(filetype.clone()),
             config_path: j.option_arg::<String, Key>(config.clone())
-                .unwrap_or_else(|| "~/.oxrc".to_string()),
+                .unwrap_or_else(|| "/opt/etc/.oxrc".to_string()),
             to_open: j.finish(),
         }
     }
