--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,7 +12,8 @@ async-trait = "0.1.83"
 base64 = "0.22.1"
 scopeguard = "1.2.0"
 
-bb8 = { version = "0.8", features = [] }
+# bb8-0.8.5: error[E0432]: unresolved import `std::sync::atomic::AtomicU64`
+bb8 = { version = "=0.8.3", features = [] }
 bytes = { version = "1.7.2", features = [] }
 clap = { version = "4.5.19", features = ["derive", "env"] }
 fast-socks5 = { version = "0.9.6", features = [] }
@@ -69,6 +70,7 @@ rcgen = { version = "0.13.1", default-fe
 
 [dev-dependencies]
 testcontainers = "0.23.1"
+chrono = "=0.4.38"
 
 [profile.release]
 lto = "fat"
@@ -81,3 +83,10 @@ strip = "symbols"
 [profile.release-with-symbols]
 inherits = "release"
 strip = false
+
+[patch.crates-io]
+# aarch64
+aws-lc-sys = { path = "../aws-lc-rs-0.21.2/aws-lc-sys" }
+# localtime/zoneinfo
+chrono = { path = "../chrono-0.4.38" }
+iana-time-zone = { path = "../iana-time-zone-0.1.61" }
