--- a/leaf/Cargo.toml
+++ b/leaf/Cargo.toml
@@ -171,7 +171,7 @@ regex = { version = "1", optional = true
 openssl = { version = "0.10", features = ["vendored"], optional = true }
 
 # Ring
-ring = { version = "0.16", optional = true }
+ring = { version = "0.17", optional = true }
 
 # TLS/rustls/QUIC
 tokio-rustls = { version = "0.24", features = ["dangerous_configuration"], optional = true }
@@ -224,7 +224,7 @@ lru_time_cache = { version = "0.11", opt
 tokio-util = { version = "0.7", default-features = false, features = ["io"], optional = true }
 
 # QUIC
-quinn = { version = "0.10", default-features = false, features = ["tls-rustls", "runtime-tokio"], optional = true }
+quinn = { version = "0.10.2", default-features = false, features = ["tls-rustls", "runtime-tokio"], optional = true }
 rustls = { version = "0.21", optional = true }
 
 # API
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,3 +18,10 @@ strip = "symbols"
 [profile.dev]
 opt-level = 0
 debug = true
+
+[patch.crates-io]
+# mips(el): error: failed to run custom build command for `ring v0.16.20`
+quinn = { path = "../../quinn-0.10.4/quinn" }
+# localtime/zoneinfo
+chrono = { path = "../../chrono-0.4.31" }
+iana-time-zone = { path = "../../iana-time-zone-0.1.59" }
