error: failed to run custom build command for `ring v0.16.20`
---

--- a/crates/arti-relay/Cargo.toml
+++ b/crates/arti-relay/Cargo.toml
@@ -52,7 +52,7 @@ tor-netdir = { path = "../tor-netdir", v
 tor-netdoc = { path = "../tor-netdoc", version = "0.28.0" }
 tor-proto = { path = "../tor-proto", version = "0.28.0", features = ["tokio"] }
 tor-relay-crypto = { path = "../tor-relay-crypto", version = "0.28.0" }
-tor-rtcompat = { path = "../tor-rtcompat", version = "0.28.0", features = ["rustls", "tokio"] }
+tor-rtcompat = { path = "../tor-rtcompat", version = "0.28.0", features = ["tokio"] }
 tracing = "0.1.36"
 tracing-subscriber = { version = "0.3.0", features = ["env-filter"] }
 
